org.mixedrealitytoolkit.spatialmanipulation/Solvers/HandConstraintPalmUp.cs

```diff
@@ -296,14 +297,14 @@ private bool IsUserGazeMeetingThresholdRequirements(XRNode hand)
                 #pragma warning restore CS0618
                 else if (TrackedPoseDriverLookup != null)
                 {
-                    InputTrackingState gazeTrackingStateInput = (InputTrackingState)TrackedPoseDriverLookup.GazeTrackedPoseDriver.trackingStateInput.action.ReadValue<int>();
```

Comment:
This line could potentially cause a NullRefException if GazeTrackedPoseDriver isn't defined or the action isn't defined, so opted for the more safe TryGetTrackingState below.

Reply:
No longer using TryGetTrackingState, just modified existing logic slightly to avoid possible NullRefException instead.